test/support/helpers/local.js

```diff
@@ -122,6 +122,7 @@ class LocalTestHelpers {
   }
 
   startSimulation() {
+    // eslint-disable-next-line no-unused-vars
     return new Promise((resolve, reject) => {
```

Comment:
We could probably lose the `reject` parameter here as well.

Reply:
I believed one of the linter was complaining about missing `reject` param, but it doesn't seem to be the case.

Reply:
Fixing everywhere.